main/io.home-assistant/air.js

```diff
@@ -0,0 +1,22 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of io.home-assistant
+//
+// Copyright 2021
```

Comment:
Copyright line should be:
```
Copyright <Year> The Board of Trustees of the Leland Stanford Junior University
```
(unless you have a different arrangement in your contract)

Reply:
ok